features/angle.yml

```diff
@@ -0,0 +1,11 @@
+name: angle
+description: The `angle` CSS type represents a portion of a circle, expressed in degrees (`deg`), gradians (`grad`), radians (`rad`), or turns (`turn`). The `angle-percentage` CSS type represents an angle or percentage value.
```

Comment:
Do we need to mention the `angle-percentage` type? If it's just that a value can be either an angle or percentage, I think we just need to describe angle here, and the `angle | percentage` union type is only important in the scope of where it's used.

Reply:
If removed here, it should be removed from the related features as well. Is there any issue removing it an leaving the key?